tests/spicy/types/unit/parseable-fail.spicy

```diff
@@ -0,0 +1,10 @@
+# @TEST-EXEC-FAIL: spicyc -c %INPUT >output 2>&1
+# @TEST-EXEC:      btest-diff output
+
+module Test;
```

Comment:
I really need to check how hard it would be to write unit tests for this as this would be a good candidate for a well-isolated C++ unit test. Checking that this fails validation seems easy, but extracting the error seems slightly harder.

Nothing we need to do about this here and now.

A naive translation could look like this
```
TEST_CASE("FOO") {
    auto source = R"(
module Test;

const Marker = "anything";

public type Header = unit {
    stuff:	Marker; # Not a parseable type
};)";
    std::stringstream input;
    input << source;
    auto node = spicy::parseSource(input, "test.spicy");
    REQUIRE(node);

    bool found_errors = false;
    spicy::detail::preTransformValidateAST(&*node, nullptr, &found_errors);
    CHECK(found_errors);

    // TODO: validate that correct error is attached to the `stuff` node (?).
}
```

Reply:
Yeah, I hear you, but I have to say that I find a btest more natural for such tests--and easier to write as well. I could maybe see this if we came up with some tooling that does all the boilerplate, so that one just needs to write that `R"` string and the expected error message. Or we just try to make the btest quicker, really shouldn't need >1secs to execute.